src/main/java/com/savvato/tribeapp/controllers/AttributesAPIController.java

```diff
@@ -41,11 +41,10 @@ public ResponseEntity<List<AttributeDTO>> getAttributesForUser(@PathVariable Lon
     public ResponseEntity<Boolean> applyPhraseToUser(@RequestBody @Valid AttributesRequest req) {
         boolean rtn = false;
 
-        if (req.noun == null || req.verb == null)
-            return ResponseEntity.status(HttpStatus.BAD_REQUEST).body(null);
-
         if (phraseService.isPhraseValid(req.adverb, req.verb, req.preposition, req.noun)) {
-            phraseService.applyPhraseToUser(req.adverb, req.verb, req.preposition, req.noun);
+            System.out.println("Controller: phrase is valid"); //////
```

Comment:
Another thought: most of the `log` messages defined in this PR will be `DEBUG`. Do a bit of googling, and be sure you have an idea when to use `INFO`, `WARNING` and `ERROR`, the other log statuses.

Reply:
Replied below with an update of all my changes. Added Loggers (warning and info). I removed everything else.  After looking through them, I felt like the remaining print statements were really just for my info as I was working through things and not necessary otherwise. I also threw an illegal exception if the verb or noun were blank (let me know if you want that to be a warning instead).